src/HttpAsyncClientDiscovery.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Http\Discovery;
+
+/**
+ * Finds an HTTP Asynchronous Client
+ *
+ * @author Joel Wurtz <joel.wurtz@gmail.com>
+ */
+class HttpAsyncClientDiscovery extends ClassDiscovery
+{
+    /**
+     * @var HttpAsyncClient
+     */
+    protected static $cache;
+
+    /**
+     * @var array
+     */
+    protected static $classes = [
+        'guzzle6' => [
+            'class'     => 'Http\Adapter\Guzzle6HttpAdapter',
+            'condition' => 'Http\Adapter\Guzzle6HttpAdapter'
```

Comment:
Missing comma for the last element (if this is copy&paste, then it is probably missing for the original discovery as well.